easybuild/easyconfigs/p/pygraphviz/pygraphviz-1.10-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'PythonBundle'
```

Comment:
Can you comment (just here in the PR, it doesn't need to end up in the EasyConfig) _why_ you changed this to PythonBundle? I don't particularly mind - if it works it works - but it might be good to log it for posterity :)

Reply:
You know what, I'll not block the merge over this. Still, would be nice to add it as a comment after the merge.

Reply:
Right now is mostly a matter of preference, basically trying to consolidate around a single easyblock for all installations of Python packages. There is an open issue about this #15639 . But it is not mandatory (yet).